bridge/server/utils/configuration.ts

```diff
@@ -306,6 +308,7 @@ export function envToConfiguration(env: { [key in EnvVar]?: string }): BridgeOpt
     mode: env.NODE_ENV,
     logging: {
       enabledComponents: env.LOGGING_COMPONENTS,
+      defaultLogLevel: Level[env.LOG_LEVEL as keyof typeof Level] ?? undefined,
```

Comment:
This allows the `LOG_LEVEL` only to be configured with uppercase `INFO`, `WARNING`, `ERROR `or `DEBUG` matching the keys in the enum `Level`. Maybe we should also allow lowercase?